apps/mobile/src/components/ModerationList/ModerationList.js

```diff
@@ -96,6 +96,8 @@ export default function ModerationList ({ forGroup, header, scrollRef, streamTyp
     }
   }, [hasMore, pending])
 
+  const handleDecisionOptionsOnChange = () => navigate('Stream', { streamType: 'proposal' })
```

Comment:
This is going to be redone anyway once this branch gets Tibet's context widget additions